extensions/ql-vscode/src/view/results/locations/NonClickableLocation.tsx

```diff
@@ -13,5 +13,6 @@ export function NonClickableLocation({ msg, locationHint }: Props) {
   if (msg === undefined) {
     return null;
   }
+  // eslint-disable-next-line github/a11y-no-title-attribute
```

Comment:
Is this something we should fix in a different PR?

Reply:
I don't think this is important enough to fix. This would need design work etc. for a feature that is probably almost never used.